TKGame/EntityManager.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TKGame
+{
+    static class EntityManager
```

Comment:
Not sure how I feel on having this be static at the moment. I feel like if we organize our entities correctly, then this doesn't need to be static, although I think we might need to discuss it in more detail